### PR TITLE
Dp 23082 add half image option for promotion pages

### PR DESCRIPTION
**Pull request by beto-aveiga**
## Description
- On Key Message template
  - Moves bg image to the bottom for mobile.
  - Reduces bg image height to 50% for large displays.

Full - half height if halfImage option is true
image

Mobile - goes below title and subtitle, for all key message variants
image

### CODE REVIEW

**Comment by beto-aveiga** [comment]
Hi @clairesunstudio ! Would you please review this PR?
I don't want to start the Drupal work for this PR unless you approve these changes.
Many thanks!

File: packages/assets/scss/03-organisms/_page-banner.scss [hunk 1]
```diff
@@ -22,10 +22,10 @@ $page-banner-height: 330px;
   .decision-tree & {
 
     h1 {
-      font-size: $fonts-2xlarge;
+      font-size: $fonts-xlarge;
```
Comment (clairesunstudio) [inline packages/assets/scss/03-organisms/_page-banner.scss]:
```suggestion
      font-size: $fonts-xlarge;
```
is there a reason that the h1 in decision tree is changed?

File: packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign~as-half-image.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+              {
+                "url": "../../assets/images/placeholder/500x300.png",
+                "width": "481"
+              }
+            ]
+          }
+        }
+      },
+      "mobileBgImage": "../../assets/images/placeholder/1600x600-lighthouse.jpg",
+      "subtitle": "From education, to healthcare and beyond, the work of more than 150 agencies across Massachusetts touches every aspect of our lives. We are a diverse team of mission-driven IT professionals supporting this vital work.",
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign~as-half-image.json]:
```suggestion
      "subtitle": "From education, to healthcare and beyond, the work of more than 150 agencies across Massachusetts touches every aspect of our lives. We are a diverse team of mission-driven IT professionals supporting this vital work.",
```

can you remove the subtitle to show a more compact version of the header block using the half image version?

**Comment by clairesunstudio** [comment]
looks good in general! I wonder if it's possible to remove the JS for this component, it's taking some time to shift the elements into the place e.g. https://mayflower.digital.mass.gov/patternlab/b/DP-23082_Add_half_image_option_for_promotion_pages/?p=pages-marketing-campaign (This could be a separate ticket to refactor this component)

I noticed a few things 

1. on certain screen sizes (tested on ~810px) there's unexpected empty space below the header image
2. > note: I removed the subtitle for the screenshot https://mayflower.digital.mass.gov/patternlab/b/DP-23082_Add_half_image_option_for_promotion_pages/?p=pages-marketing-campaign-as-half-image
Screen Shot 2021-10-26 at 3 54 48 PM

2. The half image option is only impacting the image height on desktop, the ratio of the image is mostly unchanged on smaller screens. Is that the intention? https://mayflower.digital.mass.gov/patternlab/b/DP-23082_Add_half_image_option_for_promotion_pages/?p=pages-marketing-campaign-as-half-image
Screen Shot 2021-10-26 at 4 27 25 PM


3. ~on certain screen sizes (before the main nav collapse and after the image move below the page header block), the page header background blend in with the site header background (I will check with Marissa on what to do here, maybe adding a margin-top of 1rem)~ 
> no action needed for this one for now ^

File: packages/assets/scss/03-organisms/_page-banner.scss (hunk 1 above)
Comment (beto-aveiga) [reply packages/assets/scss/03-organisms/_page-banner.scss]: I think I made a mistake here. Thanks @clairesunstudio . I will delete that.

File: packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign~as-half-image.json (hunk 2 above)
Comment (Javi-er) [reply packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign~as-half-image.json]: This was addressed by Luis.

**Review by Javi-er — APPROVED** [review]
All @clairesunstudio comments were addressed by Luis.

**Pull request merged**